machine-learning/db/test_QueryBuilder.py

```diff
@@ -83,10 +83,15 @@ def test_get_all_level_stable(self):
 
 
     def test_get_level_refactorings_count(self):
-        sqlExpected: str = "SELECT refactoring, count(*) FROM (SELECT refactoringcommit.refactoring FROM refactoringcommit WHERE refactoringcommit.level = 2 AND refactoringcommit.project_id in (select id from project where datasetName = \"integration-test\") ) t group by refactoring order by count(*) desc"
+        sqlExpected: str = "SELECT refactoring, count(*) FROM (SELECT refactoringcommit.refactoring FROM refactoringcommit WHERE refactoringcommit.level = 2 AND refactoringcommit.project_id in (select id from project where datasetName = \"integration-test\")) t group by refactoring order by count(*) desc"
```

Comment:
why the `FROM (<subquery>)` and not the query directly?

Reply:
Because, it uses another function to generate the Query and then makes some changes. I will also improve this in another PR.

Reply:
I will ignore these issues, because they don't affect the performance.